shared/folders/files/render.js.flow

```diff
@@ -25,7 +25,7 @@ export type UnlockDevice = {
 export type Props = {
   theme: 'public' | 'private',
   visiblePopupMenu: boolean,
-  popupMenuItems: Array<MenuItem>,
+  popupMenuItems: Array<MenuItem | null>,
```

Comment:
needed by flow for some reason. seems like a stricter subset should just work...